src/app/clients/clients-view/general-tab/general-tab.component.ts

```
@@ -417,4 +424,17 @@ export class GeneralTabComponent implements OnDestroy {
       return 'labels.buttons.View Closed Accounts';
     }
   }
+
+  loanProductTypeLabel(productType: string): string {
+    return LoanProductService.productTypeLabel(productType);
+  }
+
+  private processLoanAccounts(accounts: any[], productType: string): void {
+    accounts.map((account: any) => {
```

Biome (2.4.6)

[error] This callback passed to map() iterable method should always return a value.

(lint/suspicious/useIterableCallbackReturn)